### PR TITLE
Ads API v11

### PR DESCRIPTION
**Pull request by tushdante**
**Issue Type:** Feature

**Fixes / Relates To:** 

- [APE-1763](https://jira.twitter.biz/browse/APE-1763)
- [APE-1783](https://jira.twitter.biz/browse/APE-1783)
- [APE-1904](https://jira.twitter.biz/browse/APE-1904)

**Changes Included:**
- Bump version to 11
- Added support for PUT cards/unified cards; updated example; added tests
- Added new `budget_optimization` param to the campaigns module
- Added new `standard_delivery` param to the line items module
- Updated examples

### CODE REVIEW

**Comment by prijendev** [comment]
When it is planned to merge and release?

File: examples/cards.py [hunk 1]
```diff
@@ -1,29 +1,89 @@
...
 client = Client(CONSUMER_KEY, CONSUMER_SECRET, ACCESS_TOKEN, ACCESS_TOKEN_SECRET)
 
 # load the advertiser account instance
 account = client.accounts(ACCOUNT_ID)
 
-# create the card
-name = 'video website card'
-components = [{"type":"MEDIA","media_key":"13_1191948012077092867"},{"type":"DETAILS","title":"Twitter","destination":{"type":"WEBSITE", "url":"http://twitter.com/"}}]
-video_website_card = Card.create(account, name=name, components=components)
+# fetch all
```
Comment (osowskit) [inline examples/cards.py:18]: We should add a separate example for Card Creation

**Review by osowskit — APPROVED** [review]

File: examples/cards.py (hunk 1 above)
Comment (tushdante) [reply examples/cards.py:18]: Yep its been added below

**Comment by Cicci82** [comment]
Gg

**Pull request merged**